docs/customize.md

```diff
@@ -167,7 +167,7 @@ Disables gutter icons when doing multiple selections.
 ```
 
 ## Tag Plugin Settings
-Tag settings found in `bh_core.sublime-settings`.
+Tag settings found in `bh_tag.sublime-settings`.
```

Comment:
Not sure if this is intentional or if the docs were just out of date.  I found the tag settings in the bh_tag file though, and none of them were in the default bh_core settings, so they're essentially totally hidden.